content/applications/general/email_communication/email_servers.rst

```diff
@@ -252,3 +166,68 @@ Now, go to :menuselection:`Email Marketing --> Settings` and enable :guilabel:`D
 With these settings, Odoo uses the server with the lower priority for transactional emails, and the
 server here selected for mass mails. Note that in this case, the domain's Sender Policy Framework
 (SPF) records must be set to include both transactional and mass mail servers.
+
+.. _email_communication/inbound_messages:
+
+Manage inbound messages
+=======================
+
+Odoo relies on generic email aliases to fetch incoming messages.
+
+* **Reply messages** of messages sent from Odoo are routed to their original discussion thread (and
+  to the inbox of all its followers) by the catchall alias (**catchall@**).
```

Comment:
```suggestion
* **Reply messages** of messages sent from Odoo are routed to their original discussion thread (and
  to the inbox of all its followers) by the alias of the model if there is any or by the catchall alias (**catchall@**).
```

Well, not sure it's really interesting to note this but at least it's more accurate ;)

Reply:
This is better, thanks for the edit! 👍